content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
+   opportunity. The probability will no longer update automatically as the opportunity moves
+   through each stage of the pipeline.
 
-To view your unassigned leads, go to :menuselection:`CRM --> Leads --> Leads` and select the
-Unassigned filter.
+To reactivate automatic probability, click on the gear icon next to the probability percentage.
 
-.. image:: lead_scoring/unassigned-filter.png
+.. image:: lead_scoring/probability-gear-icon.png
    :align: center
-   :alt: Filtering for unassigned leads
+   :alt: The gear icon used to reactivate automatic probability on an opportunity form.
 
-.. note::
-   You can also find unassigned leads and create re-engagement campaigns using the :guilabel:`Email
-   Marketing` or :guilabel:`Marketing Automation` apps.
+Assign leads based on probability
+=================================
+
+Configure rule-based assignment
+-------------------------------
+
+*Odoo CRM* can assign leads/opportunities to sales teams and salespeople based on specified rules.
+To activate *rule-based assignment*, navigate to :menuselection:`CRM --> Configuration -->
+Settings`, and activate :guilabel:`Rule-Based Assignment`.
+
+The rule-based assignment feature can be set to run :guilabel:`Manually`, meaning an Odoo user must
+manually trigger the assignment, or :guilabel:`Repeatedly`, meaning Odoo will automatically trigger
+the assignment according to the chosen time period.
+
+To set up automatic lead assignment, select :guilabel:`Repeatedly` for the :guilabel:`Running`
+section. Then, customize how often Odoo will trigger the automatic assignment in the
+:guilabel:`Repeat every` section.
+
+.. image:: lead_scoring/rule-based-assignment.png
+   :align: center
+   :alt: The Rule-Based Assignment setting in CRM settings.
+
+If rule-based assignment is set to run :guilabel:`Repeatedly`, the assignment can still be
+triggered manually using the circular arrow icon in the :guilabel:`Rule-Based Assignment` settings
+(or using the :guilabel:`Assign Leads` button on the sales team configuration page).
+
+Configure assignment rules
+--------------------------
+
+Next, configure the *assignment rules* for each sales team and/or salesperson. These rules
+determine which leads Odoo assigns to which people. To get started, navigate to :menuselection:`CRM
+--> Configuration --> Sales Teams`, and select a sales team.
+
+On the sales team configuration form, under :guilabel:`Assignment Rules`, click on :guilabel:`Edit
+Domain` to configure the rules that Odoo uses to determine lead assignment for this sales team. The
+rules can include anything that may be relevant for this company or team, and any number of rules
+can be added.
+
+Click :guilabel:`Add Filter` to start creating assignment rules. Click on the :guilabel:`+` sign on
+the right of the assignment rule to add another line. Click on the :guilabel:`x` symbol to remove
+the line.
+
+To create an assignment rule based on the probability of success of an opportunity, click on the
```

Comment:
To create an assignment rule, based on the probability of an opportunity's success, click on the...